Doc/reference/simple_stmts.rst

```diff
@@ -533,8 +533,8 @@ The :keyword:`!yield` statement
    yield_stmt: `yield_expression`
 
 A :keyword:`yield` statement is semantically equivalent to a :ref:`yield
-expression <yieldexpr>`. The yield statement can be used to omit the parentheses
-that would otherwise be required in the equivalent yield expression
+expression <yieldexpr>`. The :keyword:`yield` statement can be used to omit the
```

Comment:
It might have been intentional here to not use markup: we don’t want a paragraph littered with the same link. So double backticks could be enough.

Reply:
Here's a link to the Devguide @bombs-kim https://devguide.python.org/documentation/style-guide/#links

As @merwok mentions there is no need to have multiple links in the same paragraph, double backticks would be preferred.